common/constraints/core

```diff
@@ -0,0 +1,5 @@
+freetype
```

Comment:
I would have thought core to be things like bootstrap=yes or things that are depends of base-system.  Thinks like freetype and harfbuzz and xorg-server are more in the snowflake category imho.